### ISSUE TITLE
[Suggestion] (Multiverse) world-specific skyboxes

### ISSUE THREAD

**Issue by 0ffz** [sp614x/optifine#2732]
[Multiverse](https://dev.bukkit.org/projects/multiverse-core) (among others) is a bukkit/spigot plugin which allows loading and moving between additional world files. 

I'm interested in whether it would be possible to add a setting in `sky.properties` to limit a skybox to only be shown if the player is located in a certain world.

`sky.properties` currently contains the following note:
> NOTE: The "world0" in the path refers to the overworld.  If there were worlds with skies (the Nether and End do not use the standard sky rendering), their files would be in ~/sky/world<world number>.

This led me to believe such a feature already existed, but I can't seem to get it working (I assume this is only for forge mods?)

If this feature is indeed already implemented, how would I go about limiting a skybox to a theoretical overworld type world file named `AnotherWorld`?

**Comment by sp614x** [comment]
Custom Sky supports modded dimensions.
Each dimensions has a unique integer id. The configuration for dimension id N should be in folder `~/sky/worldN`.
Here are the IDs of some modded dimensions: https://ftbwiki.org/Dimension

**Comment by 0ffz** [comment]
Makes sense. So how plausible would it be to allow this to work with world names instead of just dimensions? Is there any information the server sends on the world name itself that could be used?

It sounds like an optional setting in the properties file would indeed be the way to go, as the point would be to use this for more than one world per dimension.

**Comment by TheArcaneBrony** [comment]
Pretty sure this won't work with multiverse since that's server side and it reports as ID 0 for overworld worlds as far as i know
